src/library_html5.js

```diff
@@ -315,6 +315,7 @@ var LibraryHTML5 = {
   // Users can also add more special event targets, basically by just doing something like
   //    specialHTMLTargets["!canvas"] = Module.canvas;
   // (that will let !canvas map to the canvas held in Module.canvas).
+  $specialHTMLTargets__docs: '/** @type {Object} */',
```

Comment:
I find this a little confusing since it is delared as an array, but I guess our usage of it is as an object... JS is confusing.

Reply:
It took a little working out, and yep, it’s confusing but how it’s used sort of makes sense. For JS at least!